server/app/services/export/QuestionJsonSampler.java

```diff
@@ -184,7 +181,7 @@ public void addSampleData(ApplicantData applicantData, ApplicantQuestion applica
           /* latitude= */ 44.0462,
           /* longitude= */ -123.0236,
           /* wellKnownId= */ 4326L,
-          /* serviceArea= */ "springfield_county_InArea_1709069741,portland_NotInArea_1709069741");
+          /* serviceArea= */ "springfieldCounty_InArea_1709069741,portland_NotInArea_1709069741");
```

Comment:
The underscore was causing issues deserializing, and now that we use the correct accessor methods we get back an actual list of `ServiceAreaInclusion`s we have to re-serialize for the API.

Reply:
Weird

Reply:
Before the tests were just inserting a fake string directly into the `ApplicantData` and we were pulling it out as a string, so we were never having to serialize or deserialize anything. Now we pull it out as the correct type so we have to be able to deserialize it.